### ISSUE TITLE
Popup Customizer seems broken or not loading CSS properly.

### ISSUE THREAD

**Issue by tevyaw** [ThemeAvenue/BetterOptin#35]
Here's what I'm seeing: https://drive.google.com/file/d/0B6-_TRezO9iOY2FCSWJRb1ZicXc/view

**Comment by tevyaw** [comment]
Now that I'm looking at the old version the whole "Choose a Template" section of the Add New Popup CPT editor, isn't loading correct CSS. It looks much nicer with 3 columns on the old one. the new one it's a single column that's hard to tell which select/customize buttons go with which design, and without the checkmark overlay that the old one did (which is much nicer). Maybe it's on purpose, but hopefully not.

**Comment by tevyaw** [comment]
Okay, so the popup may have been working, contrary to what I said in the screencast. I realized Varnish was probably caching the site from before. I purged Varnish and now it's working. BUT I'd already switched back to the old version of the plugin, for now.

**Comment by SiamKreative** [comment]
Hi @tevyaw 

Making a screencast requires some time and I would like to thank you for your efforts :) That's very nice of you.

Are you still seing the issue? if yes, please provide as much details as possible about your setup (server, os, version, plugins installed). Alternatively, I could login to your site and check things.

Cheers

**Comment by tevyaw** [comment]
Screencastify (Chrome extension) makes screencasts pretty easy. Often easier than trying to explain everything in words, or even get a screenshot or two.

I immedately went back to the WP.org stable version of the plugin. So I couldn't tell you if that site is having that issue any longer. I can get you access to it, if you'd like to test there. What email should I use for the user?

**Comment by tevyaw** [comment]
I just went and purchased a license, then upgraded the plugin. Next I input the license and it says it's verified. However, it won't let me change to anything other than WordPress for the emailing provider. I tried submitting a ticket via my betteropt.in account, and it says "You are not allowed to submit a ticket." 

Can you help, and soon, please?

But it all appears to be working correctly, so that's good. I just need my MailPoet integration.

**Comment by tevyaw** [comment]
So right after my previous message I saw there was a 2.0.1 update available. I updated that, and it disabled the plugin. I activated it and everything seemed fine, including the MailPoet option (as well as MailChimp and Aweber). However, now the pop-over is not appearing on the front-end of the site (it was after the version 2.0.0 update - but that was before it disabled the plugin, so I'm assuming it wasn't fully updated yet). 

Any ideas why it's not working now? http://leadinglds.com/
